src/models/behaviors.js

```diff
@@ -13,6 +13,8 @@ var helpers = require('../util/helpers'),
     combinators = require('../util/combinators'),
     xpath = require('./xpath'),
     jsonpath = require('./jsonpath'),
+    csvToObject = require('csv-to-object'),
+    hashmap = require('hashmap'),
```

Comment:
As far as I can tell, everything you're adding to the map is a string, in which case a standard JavaScript object works fine. Is there any additional value to adding hashmap?